lua/neo-tree/defaults.lua

```diff
@@ -406,6 +406,7 @@ local config = {
       ["A"] = "add_directory", -- also accepts the config.show_path and config.insert_as options.
       ["d"] = "delete",
       ["r"] = "rename",
+      ["b"] = "rename_basename",
```

Comment:
Can you also add it here in the README as well: https://github.com/nvim-neo-tree/neo-tree.nvim/blob/395ce4675a54828f9fd730d78f7910c114ab0a49/README.md?plain=1#L286